app/assets/javascripts/hyrax/editor/controlled_vocabulary.es6

```diff
@@ -114,9 +128,25 @@ export default class ControlledVocabulary extends FieldManager {
   // '_destroy' hidden parameter
   removeFromList( event ) {
       event.preventDefault()
+      // Changing behavior of the remove button to add a new field if the last field is removed
+      // Using querySelector to find elements with data-attribute="based_near"
+      const basedNearElements = document.querySelectorAll('[data-attribute="based_near"]');
```

Comment:
I think we should generalize this selection to not be based_near specific. I know it's the only thing its being used for at the moment, but I am guessing we'd need to do that anyways before sharing back with hyrax and might be helpful for ROR.

If i'm understanding correctly, i think this would have the same affect:
`this.element.find('input.' + this.inputTypeClass)`
Plus we'd probably want to rename some variables to not specifically name basedNear.